src/pages/index.js

```diff
@@ -7,20 +7,20 @@ import LabelTitle from '../Components/Label';
 import ApplyLink from '../Components/Link';
 import Quote from '../Components/Quote';
 import Title from '../Components/Title';
-import Avatar from '../components/avatar'
-
+import Avatar from '../Components/avatar'
+// test comment
```

Comment:
Unnecessary comment, can you please delete it?

Reply:
sure, it was a test commit to check how pull request work. Deleting it now.